plugins/backstage-plugin-coder/src/hooks/reactQueryWrappers.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file Defines a couple of wrappers over React Query/Tanstack Query that make
+ * it easier to use the Coder SDK within UI logic.
```

Comment:
Do most Backstage users use React Query?  I worry we are abstracting too early.

If we just expose the base `useCoderSdk` and let users hook it up however they need, that seems most ideal to me.

Reply:
I'm sure some of them do, but Backstage's default recommendations don't include it

I cover this in the README PR, but there are a lot of limitations around `useEffect`/`useAsync`, so I figured it would help to provide an easier way to bring RQ into the app. The plugin is still going to expose the SDK directly for people who want to wire things up in a more custom way